specification/electronic-prescription-service-api.yaml

```diff
@@ -18,12 +18,11 @@ info:
     ### As a prescriber
     You can:
     * encode prescription data
-    * create a secondary care (outpatient) prescription
-    * cancel a secondary care (outpatient) prescription
+    * create and cancel a secondary care (outpatient) prescription
+    * create and cancel a primary care prescription
 
     You cannot currently use this API to:
-    * create a primary care prescription
-    * create a secondary care (other than outpatient) prescription
+    * create a or cancel secondary care (homecare) prescription
```

Comment:
Same as above, separate bullets? 
Also, an extra "a" in "create or cancel"

Reply:
I combined them in maybe a failed attempt to shorted the document slightly as this top section is so long! Can split them again if we think its better?